src/components/Table.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { connect } from 'react-redux';
+
+class Table extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.handleMove = this.handleMove.bind(this)
+    this.win = this.win.bind(this);
+  }
+
+  handleMove(index) {
```

Comment:
it should dispatch `MOVE` and send `index` in the payload. The aim is to learn redux not only in terms of dispatching events. Suggestion: move logical part to `redux`